go.mod

```diff
@@ -20,6 +20,7 @@ require (
 	github.com/valyala/gozstd v1.7.0
 	github.com/valyala/histogram v1.0.1
 	github.com/valyala/quicktemplate v1.4.1
+	go.mongodb.org/mongo-driver v1.3.2
```

Comment:
I'm afraid this violates the following principle - https://medium.com/@valyala/stripping-dependency-bloat-in-victoriametrics-docker-image-983fb5912b0d :(

What is the reason to include it here?

Reply:
go build -ldflags "-s -w" main.go, vmagent-prod size keeps 12M, same with last version prod

I am trying to do a hash for sw, to check whether it changes, of course I can make a []byte manually using sew struct, but seems a little ugly, so I use bson encode

bson comes from mongo-driver, I will think about make it more light

Reply:
I see. It is still better to write or copy-paste small code snippets from external package instead of importing it. See https://www.youtube.com/watch?v=PAAkCSZUG1c&t=9m28s and https://news.ycombinator.com/item?id=14903059 for details.